lib/registration.pm

```diff
@@ -223,16 +224,32 @@ sub ssh_add_suseconnect_product {
 
     remove_suseconnect_product($name, [$version, [$arch, [$params]]]);
 
-Wrapper for SUSEConnect -d $name.
+wrapper for suseconnect -d $name.
 =cut
 sub remove_suseconnect_product {
     my ($name, $version, $arch, $params) = @_;
     $version //= scc_version();
-    $arch //= get_required_var('ARCH');
+    $arch //= get_required_var('arch');
```

Comment:
Why is this now lowercase?

Reply:
this question remain unanswered , can you please clarify ?

Reply:
Yes, in `vim`, if you select some text, you can press `y` to yank it or `u` to make it lowercase. I simply didn't notice I pressed `u` instead of `y`.